operator/controllers/csiscaleoperator_controller.go

```diff
@@ -530,29 +539,71 @@ func (r *CSIScaleOperatorReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		return watchResources[resourceKind][resourceName]
 	}
 
+	restartCSIPodsOnCreateDelete := func(cfgmapData map[string]string) bool {
+		restart := false
+		for key := range cfgmapData {
+			if !strings.HasPrefix(strings.ToUpper(key), "VAR_DRIVER_") {
```

Comment:
Minor suggestion - "VAR_DRIVER_" is used in multiple places, please make it a const (with a name something like driverENVPrefix?)